From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Mar 2023 23:34:50 +0000
Subject: driver: Set a non-trivial window title

When running in "existing user" mode (which isn't done upstream, but
Debian and Ubuntu re-enable it because they don't have GNOME Tour),
the window title is visible in Alt+[key above Tab] and the Overview.
It defaults to "gnome-initial-setup", but looks better if we reuse the
translated name from the .desktop file.

Signed-off-by: Simon McVittie <smcv@debian.org>
(cherry picked from commit 2ed2453476bc62501896083109c542cf407c7ae1)
---
 gnome-initial-setup/gis-driver.c | 1 +
 po/POTFILES.in                   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/gnome-initial-setup/gis-driver.c b/gnome-initial-setup/gis-driver.c
index 9526e62..2e5fdf1 100644
--- a/gnome-initial-setup/gis-driver.c
+++ b/gnome-initial-setup/gis-driver.c
@@ -813,6 +813,7 @@ gis_driver_startup (GApplication *app)
                                     "application", app,
                                     "icon-name", "preferences-system",
                                     "deletable", FALSE,
+                                    "title", _("Initial Setup"),
                                     NULL);
 
   g_signal_connect (driver->main_window,
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 96ed603..4ec8809 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -4,6 +4,7 @@ data/gnome-initial-setup-first-login.desktop.in.in
 data/gnome-initial-setup.desktop.in.in
 gnome-initial-setup/cc-common-language.c
 gnome-initial-setup/gis-assistant.c
+gnome-initial-setup/gis-driver.c
 gnome-initial-setup/gnome-initial-setup.c
 gnome-initial-setup/pages/account/gis-account-avatar-chooser.ui
 gnome-initial-setup/pages/account/gis-account-page-enterprise.c
